docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -630,6 +630,12 @@ Set this flag to `true` to enable audit logging for the universe.
 
 For details, see [Audit logging for the YCQL API](../../../secure/audit-logging/audit-logging-ycql).
 
+##### --ycql-allow-non-authenticated-password-reset
+
+Set this flag to `true` to enable a superuser to reset a password.
```

Comment:
Not clear - from instructions on the other page, do I not need to set --use_cassandra_authentication to false? 
If so, mention all caveats here.
Or are the instructions on the other page more of a suggestion when they switch authentication off?

Reply:
good point. Yes we need to mention it.